### ISSUE TITLE
Title of the website without <br>

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#252]
![image](https://cloud.githubusercontent.com/assets/7644740/4441281/d49a22e8-47d0-11e4-9e36-7b008e6181ab.png)
The problem was corrected for the global navigation, but there is still a `<br>` in the title, and screenreaders interpret this as 2 links + check if there are not other elements with `<br>` as they all should be removed.
